template/template.go

```diff
@@ -32,6 +32,8 @@ import (
 	"github.com/prometheus/client_golang/prometheus"
 	"github.com/prometheus/common/model"
 
+	commonTemplates "github.com/prometheus/common/helpers/templates"
```

Comment:
I don't think there's any convention, but I haven't seen any camelCase alias so far, I'd go with `common_templates` or `common_template` to align with text_template and html_template.

Reply:
Fixed.